### ISSUE TITLE
does not comply with psr-4 autoloading standard

### ISSUE THREAD

**Issue by pbksol** [kimai/Paid-plugins#184]
I put it in here because it seems to only affect (older) plugin migrations…

I know it is something related to Laravel and changes with case sensitive stuff (didn't dove deeper and don't get why they changed that). But… I'm quite confident it's already known but just to allow people to find it…

`Class ExpensesBundle\Migrations\Version20220320125148 located in ./var/plugins/ExpensesBundle/Migrations/Version20220320125148.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class ExpensesBundle\Migrations\Version20210706114934 located in ./var/plugins/ExpensesBundle/Migrations/Version20210706114934.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class ExpensesBundle\Migrations\Version20210727140528 located in ./var/plugins/ExpensesBundle/Migrations/Version20210727140528.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class ExpensesBundle\Migrations\Version20200130122003 located in ./var/plugins/ExpensesBundle/Migrations/Version20200130122003.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class MetaFieldsBundle\Migrations\Version20190916141013 located in ./var/plugins/MetaFieldsBundle/Migrations/Version20190916141013.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class MetaFieldsBundle\Migrations\Version20210223154013 located in ./var/plugins/MetaFieldsBundle/Migrations/Version20210223154013.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class MetaFieldsBundle\Migrations\Version20190916163556 located in ./var/plugins/MetaFieldsBundle/Migrations/Version20190916163556.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class MetaFieldsBundle\Migrations\Version20200828153148 located in ./var/plugins/MetaFieldsBundle/Migrations/Version20200828153148.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class MetaFieldsBundle\Migrations\Version20190917165134 located in ./var/plugins/MetaFieldsBundle/Migrations/Version20190917165134.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class MetaFieldsBundle\Migrations\Version20191231190117 located in ./var/plugins/MetaFieldsBundle/Migrations/Version20191231190117.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class AuditTrailBundle\Migrations\Version20200327165908 located in ./var/plugins/AuditTrailBundle/Migrations/Version20200327165908.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class AuditTrailBundle\Migrations\Version20190903141012 located in ./var/plugins/AuditTrailBundle/Migrations/Version20190903141012.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class AuditTrailBundle\Migrations\Version20210118201925 located in ./var/plugins/AuditTrailBundle/Migrations/Version20210118201925.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class AuditTrailBundle\Migrations\Version20210118201926 located in ./var/plugins/AuditTrailBundle/Migrations/Version20210118201926.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class AuditTrailBundle\Migrations\Version20231202214543 located in ./var/plugins/AuditTrailBundle/Migrations/Version20231202214543.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class TaskManagementBundle\Migrations\Version20200914152531 located in ./var/plugins/TaskManagementBundle/Migrations/Version20200914152531.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class TaskManagementBundle\Migrations\Version20993112235959 located in ./var/plugins/TaskManagementBundle/Migrations/Version20993112235959.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class TaskManagementBundle\Migrations\Version20191207120554 located in ./var/plugins/TaskManagementBundle/Migrations/Version20191207120554.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class TaskManagementBundle\Migrations\Version20240416172700 located in ./var/plugins/TaskManagementBundle/Migrations/Version20240416172700.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.
Class TaskManagementBundle\Migrations\Version20210131151530 located in ./var/plugins/TaskManagementBundle/Migrations/Version20210131151530.php does not comply with psr-4 autoloading standard (rule: KimaiPlugin\ => ./var/plugins). Skipping.`

**Comment by kevinpapst** [comment]
Yeah, its a warning that Composer recently introduced. 

Its annoying and unfortunately unfixable for now 🤷 😢

**Comment by pbksol** [comment]
Yeah, that's what I thought :-)

We can close this issue or leave it open – whatever you like. Other people searching for it can now find something about it.

It might scare some people just because there's very much yellow showing up on the screen 😂

**Comment by kevinpapst** [comment]
Yeah, super annoying, right?! I knew the second I saw it, that I will have multiple discussions about it 😁 

Let's leave it open, for the next user searching.
